assets/javascripts/discourse/components/bar-chart.js.es6

```diff
@@ -0,0 +1,97 @@
+import loadScript from "discourse/lib/load-script";
+import { default as computed } from "discourse-common/utils/decorators";
+import themeColor from "../lib/themeColor";
+
+export default Ember.Component.extend({
+  barsColor: themeColor("--tertiary"),
+  barsHoverColor: themeColor("--tertiary-high"),
+  gridColor: themeColor("--primary-low"),
+  labelsColor: themeColor("--primary-medium"),
+  chart: undefined,
```

Comment:
I would prefer null here, unless you have a specific reason to use undefined ?

Reply:
Definitely `null` is better for an absent object.